src/runner.ts

```diff
@@ -146,11 +146,14 @@ export class Runner {
         let program: ts.Program | undefined;
 
         if (this.options.project != null) {
-            if (!fs.existsSync(this.options.project)) {
+            let project: string;
+            try {
+                project = findTsconfig(this.options.project);
```

Comment:
I think this function should handle error handling itself (and return `string | undefined` instead of throwing an error that gets ignored.